tools/dataProtector/dataProtectorCore/protectData.md

```diff
@@ -153,14 +153,14 @@ const protectedData = await dataProtectorCore.protectData({
 
 You can expect this callback function to be called with the following titles:
 
-```
-'EXTRACT_DATA_SCHEMA'
-'CREATE_ZIP_FILE'
-'CREATE_ENCRYPTION_KEY'
-'ENCRYPT_FILE'
-'UPLOAD_ENCRYPTED_FILE'
-'DEPLOY_PROTECTED_DATA'
-'PUSH_SECRET_TO_SMS'
+```ts
+'EXTRACT_DATA_SCHEMA';
+'CREATE_ZIP_FILE';
+'CREATE_ENCRYPTION_KEY';
+'ENCRYPT_FILE';
+'UPLOAD_ENCRYPTED_FILE';
+'DEPLOY_PROTECTED_DATA';
+'PUSH_SECRET_TO_SMS';
```

Comment:
Hum that's not really TypeScript here.
Why not keep it the way it was?

Reply:
i had some warning on my side with the linter as there was not language set